src/coprocessor/codec/batch/lazy_column_vec.rs

```diff
@@ -329,7 +330,7 @@ mod tests {
                     .unwrap();
                 assert_eq!(col.len(), 2);
                 assert_eq!(col.eval_type(), EvalType::Real);
-                assert_eq!(col.as_real_slice(), &[Some(1.0), None]);
+                assert_eq!(col.as_real_slice(), &[Real::new(1.0).ok(), None]);
```

Comment:
Can we provide an extra function to return `Option` directly, and to avoid this annoying form?

Reply:
We can use our own Real, wraps the ordered-float. I didn't do this because I think adding `.ok()` is already simple enough.